bmaptools/BmapHelpers.py

```diff
@@ -83,3 +94,72 @@ def program_is_available(name):
             return True
 
     return False
+
+
+def get_mount_point(filepath):
+    """
+    Return the mount point the file object file resides on.
+    """
+    f = os.path.realpath(filepath)
```

Comment:
Pylint complains about one-letter variables. Call it "part" or something, something along the lines of "partial path", or "path prefix", or "path component". But "f" is not a good choice for a variable name. Have it self-documenting, at least e little bit.